docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -12,173 +14,630 @@ The syntax of `MATCH` is relatively more flexible compared with that of other qu
+Got 3 rows (time spent 3107/3683 us)
+```
+
+> **OpenCypher compatibility**
+>
+> For now, nGQL 2.0 only supports string-type VIDs.
+
+### Match connected vertices
+
+You can use the `--` symbol to represent edges of both directions and match vertices connected by these edges.
+
+> **nGQL compatibility**
+>
+> * In nGQL 1.x, the `--` symbol is used for inline comments.
+> * Starting from nGQL 2.0, the `--` symbol represents an incoming or outgoing edge.
+
+```nGQL
+nebula> MATCH (v:player{name:"Tim Duncan"})--(v2) RETURN v2.name AS Name;
++---------------------+
+| Name                |
++---------------------+
+| "Tony Parker"       |
++---------------------+
+| "LaMarcus Aldridge" |
++---------------------+
+| "Marco Belinelli"   |
++---------------------+
+| "Danny Green"       |
++---------------------+
+| "Aron Baynes"       |
++---------------------+
+...
+Got 13 rows (time spent 6029/8976 us)
+```
+
+And you can add a `>` or `<` symbol to specify the direction of the edge.
+
+```nGQL
+nebula> MATCH (v:player{name:"Tim Duncan"})-->(v2) RETURN v2.name AS Name;
++-----------------+
+| Name            |
++-----------------+
+| "Spurs"         |
++-----------------+
+| "Tony Parker"   |
++-----------------+
+| "Manu Ginobili" |
++-----------------+
+Got 3 rows (time spent 2897/5993 us)
+```
+
+To extend the pattern, add more edges and vertices.
+
+```nGQL
+nebula> MATCH (v:player{name:"Tim Duncan"})-->(v2)<--(v3) RETURN v3.name AS Name;
++---------------------+
+| Name                |
++---------------------+
+| "Tony Parker"       |
++---------------------+
+| "Tiago Splitter"    |
++---------------------+
+| "Dejounte Murray"   |
++---------------------+
+| "Tony Parker"       |
++---------------------+
+| "LaMarcus Aldridge" |
++---------------------+
+...
+```
+
+If you don't need to refer to a vertex, you can omit the variable representing it in the parentheses.
+
+```nGQL
+nebula> MATCH (v:player{name:"Tim Duncan"})-->()<--(v3) RETURN v3.name AS Name;
++---------------------+
+| Name                |
++---------------------+
+| "Tony Parker"       |
++---------------------+
+| "LaMarcus Aldridge" |
++---------------------+
+| "Rudy Gay"          |
++---------------------+
+| "Danny Green"       |
++---------------------+
+| "Kyle Anderson"     |
++---------------------+
+...
+```
+
+### Match paths
+
+Connected vertices and edges form a path. You can use a user-defined variable as follows to name a path.
+
+```nGQL
+nebula> MATCH p=(v:player{name:"Tim Duncan"})-[]->(v2) RETURN p;
++-----------------------------------------+
+| p                                    |
++-----------------------------------------+
+| ("player100")-[:serve@0]->("player204") |
++-----------------------------------------+
+| ("player100")-[:like@0]->("player101")  |
++-----------------------------------------+
+| ("player100")-[:like@0]->("player125")  |
++-----------------------------------------+
+Got 3 rows (time spent 3717/4573 us)
+```
+
+### Match edges
```

Comment:
should this section be moved before "Match paths"? what the [] symbol represents is described in this section, but it appears in the "Match paths" section.

Reply:
Modified the statement in the **MATCH paths** section to `MATCH p=(v:player{name:"Tim Duncan"})-->(v2) RETURN p`. The `[]` symbols are ommited. 
The return message still has edge information, but that's OK because the next section is about edges.